docs/developer-guide.md

```diff
@@ -39,34 +61,3 @@ In order to avoid potential side effects using `lerna version/publish` commands,
    ```sh
        yarn sync-version --name=<package-name>
    ```
-
-# Setup
-
-- Clone this repository
-- Run `yarn` to install package dependencies
-- Run `yarn build` to compile all packages
-- Run `yarn start` in an example app to test
-
-```sh
-yarn bootstrap
-yarn llink
-yarn build
-```
-
-## Storybook
-
-To run locally, run `yarn llink` followed by `yarn storybook:<theme>` where theme is one of:
-
-- base
-- gov
-
-To build, run `yarn llink` followed by `yarn build-storybook:<theme>` where theme is one of:
-
-- base
-- gov
-
-To test build locally, run `npx http-server ./storybook-static` after building.
```

Comment:
Not sure these were about. Style changes hot reload without needing to build. But when I tried making a new component I couldn't get it to show in Storybook.